completions/bash/kpod

```diff
@@ -222,18 +223,20 @@ _kpod_mount() {
 
 _kpod_push() {
     local boolean_options="
-     --disable-compression
-     -D
-     --quiet
-     -q
-     --signature-policy
-     --certs
-     --tls-verify
-     --remove-signatures
-     --sign-by
+    --disable-compression
```

Comment:
nit.  alphabetize these entries and the next ones..

Reply:
fixed